bliss/surveys/dc2.py

```diff
@@ -262,33 +382,43 @@ def from_file(cls, cat_path, wcs, height, width, band):
             "star_log_fluxes": star_log_fluxes.reshape(1, nobj, 6),
         }
 
-        return cls(height, width, d), torch.stack(psf_params)
+        if return_match_id:
```

Comment:
Can we always return match_id? A return type that varies in dimension is something of an anti-pattern.